schema/base-model.js

```diff
@@ -66,9 +66,9 @@ class BaseModel extends Model {
       .whereNot(`${this.tableName}.deleted`, null);
   }
 
-  static count(establishmentId, query) {
+  static count(value, query, key = 'establishmentId') {
```

Comment:
This is changing the function signature, do we use this in any of the models that inherit from it?

Reply:
See [comments above](https://github.com/UKHomeOffice/asl-schema/pull/444#discussion_r586613171).